src/components/TextField.tsx

```diff
@@ -39,19 +45,38 @@ export const TextField: FC<FieldProps<TextFieldSchema>> = ({
     renderAfter,
     disabled,
     readOnly,
+    copyToClipboard,
+    validateOnChange,
+    loadingValidate,
   } = field;
 
   const { isReadOnly } = useContext(Ctx);
 
   const fieldStyles = useStyles<FieldStyles>('textField', styles);
 
-  const { register, control } = useFormContext();
+  const { register, control, watch } = useFormContext();
 
   const errorMessage = useErrorMessage(name, label);
 
   const values = useWatch({
     control,
   });
+  const value = watch(name);
```

Comment:
If we have values above, from useWatch, we should be able to derive the value given the name from that